products/swanbot.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title:  SwanBot
+categories: swancomponents
+tab-id: swan-panel-one
+thumbnail: images/swanbot.png
+order: 1
+---
+
+<div class='swan-flex-col'>
```

Comment:
Can we use Markdown instead of inline HTML?

Reply:
Removed inline html and added the details as front matter in each of the files. https://github.com/subnero1/subnero1.github.io/pull/203/commits/cfdf890e8790e300127ec22c1c810f0336c16acd